sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -1081,12 +1021,44 @@ public Builder setNameBytes(
         // @@protoc_insertion_point(builder_scope:sonarqube.ws.rules.ListResponse.Rule)
       }
 
+      // @@protoc_insertion_point(class_scope:sonarqube.ws.rules.ListResponse.Rule)
+      private static final org.sonarqube.ws.Rules.ListResponse.Rule DEFAULT_INSTANCE;
       static {
-        defaultInstance = new Rule(true);
-        defaultInstance.initFields();
+        DEFAULT_INSTANCE = new org.sonarqube.ws.Rules.ListResponse.Rule();
+      }
+
+      public static org.sonarqube.ws.Rules.ListResponse.Rule getDefaultInstance() {
+        return DEFAULT_INSTANCE;
+      }
+
+      public static final com.google.protobuf.Parser<Rule> PARSER =
+          new com.google.protobuf.AbstractParser<Rule>() {
+        public Rule parsePartialFrom(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)